lib/quibble.mjs

```diff
@@ -80,8 +99,8 @@ function getStubsInfo (moduleUrl) {
   if (!moduleUrl.searchParams.get('__quibble')) return undefined
 
   const moduleFilepath = moduleUrl.pathname
-
-  return [...global.__quibble.quibbledModules.entries()].find(([m]) => m === moduleFilepath)
+  const stub = global.__quibble.quibbledModules.get(moduleFilepath)
+  return stub ? [moduleFilepath, stub] : undefined
```

Comment:
drive-by performance improvement

Reply:
always appreciated. this lib doesn't get a lot of love